tests/helpers.py

```diff
@@ -395,153 +394,103 @@ def validate_cephfilesystem(fs_name):
     )
     ct_pod = pod.get_ceph_tools_pod()
     ceph_validate = False
-    k8s_validate = False
+    ocp_validate = False
     cmd = "ceph fs ls"
-    logger.info(fs_name)
+
+    result = CFS.get(resource_name=fs_name)
+    if result.get('metadata').get('name'):
+        logger.info("Filesystem %s got created from Openshift Side", fs_name)
+        ocp_validate = True
+    else:
+        logger.info(
+            "Filesystem %s was not create at Openshift Side", fs_name
+        )
+        return False
+
     out = ct_pod.exec_ceph_cmd(ceph_cmd=cmd)
     if out:
         out = out[0]['name']
-        logger.info(out)
         if out == fs_name:
-            logger.info("FileSystem got created from Ceph Side")
+            logger.info("FileSystem %s got created from Ceph Side", fs_name)
             ceph_validate = True
         else:
-            logger.error("FileSystem was not present at Ceph Side")
+            logger.error("FileSystem %s was not present at Ceph Side", fs_name)
             return False
-    result = CFS.get(resource_name=fs_name)
-    if result['metadata']['name']:
-        logger.info(f"Filesystem got created from kubernetes Side")
-        k8s_validate = True
-    else:
-        logger.error("Filesystem was not create at Kubernetes Side")
-        return False
-    return True if (ceph_validate and k8s_validate) else False
+    return ceph_validate and ocp_validate
```

Comment:
let's return only one bool here, True in case the FS exists in both ocp and ceph, False otherwise

Reply:
Done